samza-core/src/main/java/org/apache/samza/clustermanager/AbstractContainerAllocator.java

```diff
@@ -55,6 +55,8 @@
    */
   private final TaskConfig taskConfig;
 
+  private final ClusterManagerConfig clusterManagerConfig;
```

Comment:
this should be no longer necessary right? since, we've delegated allocator-specific behavior to the respective sub-classes

Reply:
Oops removed, good catch!